onset_maproom/config-anacim-senegal.yaml

```diff
@@ -11,6 +11,19 @@ listen_port: 8063
 dev_processes: 4  # number of server processes in dev setup. Not used when running in apache.
 #Jeff set up host 0.0.0.0 port 8000 to run app on  http://shortfin01.iri.columbia.edu:8000/onset-maproom/ for demos
 
+dbpool:
+    name: iridb  # postgres database name
+    host: devi.iri.columbia.edu
```

Comment:
Did you add new tables to devi to support this project?

Could you try changing this to use dlcomputemon1 instead of devi? The db on devi was part of the plan for upgrading postgres, but the plan has changed, and I would like to avoid adding new clients of devi because eventually we need to turn it off.